code-style/scss/README.md

```diff
@@ -1,52 +1,257 @@
-```css
-.module {}
-/* or */
-.even-grid {}
+1. [Introduction](#Introduction)
+2. [Class Naming Theory](#Class-Naming-Theory)
+3. [BEM](#BEM)
+  1. [Using BEM](#Using-BEM)
+    1. [Block](#Block)
+    2. [Element](#Element)
+    3. [Modifier](#Modifier)
+  2. [BEM in Sass](#BEM-in-Sass)
+4. [ITCSS](#ITCSS)
+  1. [File Naming](#File-Naming)
+    1. [Settings](#Settings)
+    2. [Tools](#Tools)
+    3. [Generic](#Generic)
+    4. [Elements](#Elements)
+    5. [Objects](#Objects)
+    6. [Components](#Components)
+    7. [Utilities](#Utilities)
+5. [Name Spacing](#Name-Spacing)
+6. [ITCSS and BEM](#ITCSS-and-BEM)
+
+
+## Introduction
+
+Our CSS is organized using [Harry Roberts](csswizardry.com)' Inverted Triangle CSS (ITCSS) organizational approach. This method is mixed with [Block Element Modifier](http://getbem.com) (BEM) naming convention for class names throughout the stylesheet.
+
+---
+
+## Class Naming Theory
+
+When it comes to naming classes, we find it best to be as specific as possible with the fewest words possible. This can be an exercise in understanding the purpose and function of a component. It's helpful to find like components and think about what they have in common and what their purpose is. You can then choose a generic name that encapsulates the uses the component will have.
+
+Keep in mind that naming is for humans, so verboseness is useful if it helps communicate what the class does. Ideally, a future developer should be able to look at a class name and have a reasonable idea of what they might see on the rendered page.
+
+---
+
+## BEM
+
+BEM stands for block element modifier, which is the order you follow for naming with BEM.
+
+#### Block
+
+`.block` is the parent wrapper of the component. This would be something like site navigation, the hero section, or a footer. Any container that holds other code could be a block.
+
+#### Element
+
+`.block__element` is an element inside the component, or inside the parent wrapper. This could be a title or a link. It's anything inside the parent wrapper. It's important here to note that BEM isn't meant to reflect nesting. For example, say you have a section that includes an image element inside an anchor element:
+
+```html
+<section>
+  <a href="#">
+    <img src="#">
+  </a>
+</section>
 ```
 
-**Subcomponents** are bits and pieces of a module that are specific to that module. A module is separated from a subcomponent with *double--hyphens*:
-```css
-.module--subcomponent {}
-/* or */
-.even-grid--item {}
+Our BEM naming doesn't need to show that we nested that image tag, so we would give it a class name of something like `.block__image`. We would _not_ give it a class name of `.block__link__image`.
+
+#### Modifier
+
+Modifiers are the most specific of the three parts of BEM. They should only define one or two properties that change the previously defined properties on the block or element. You can use a modifier on a block or an element.
```

Comment:
The way this paragraph starts is inconsistant with the other two. Though, I like the syntax here, where you use the name "Modifier" as a word and show the syntax later on - it's less confusing.